internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-nlb-dev.stack.yml

```diff
@@ -474,7 +473,7 @@ Resources: # If a bucket URL is specified, that means the template exists.
               - "/*"
       ListenerArn: !GetAtt EnvControllerAction.HTTPSListenerArn
       Priority: !GetAtt HTTPSRulePriorityAction.Priority
-  PublicNetworkLoadBalancer:
+  PublicNetworkLoadBalancerV2:
```

Comment:
Could you remind me again: is this because otherwise the resource update (replacement) will fail?

Reply:
Yes